R/user_interactions.R

```diff
@@ -1,3 +1,6 @@
+readline <- NULL
```

Comment:
does this come from R CMD check or lintr? The linter for unused objects can be a bit finicky (I often end up ignoring it).

Reply:
I put this here because I am mocking a base R function within my unit tests. 
It is from this guidance: https://testthat.r-lib.org/reference/local_mocked_bindings.html

Reply:
aaah ok!